WzComparerR2.MapRender/UI/UIWorldMap.cs

```diff
@@ -114,6 +111,15 @@ protected override void InitializeComponents()
             canvas.Children.Add(cmbQuestList);
             this.CmbQuestList = cmbQuestList;
 
+            ComboBox cmbFogList = new ComboBox();
+            cmbFogList.Width = 100;
+            cmbFogList.Height = 20;
+            Canvas.SetLeft(cmbFogList, 250);
```

Comment:
`cmbFogList` is coincides with `cmbQuestList`, is there an worldmap have both of them?

Reply:
Maybe not. In CMS 186, there was not any WorldMap that has QuestLimit.

Reply:
looks good for me. maybe we can use stackpanel later for more options in future.